.github/workflows/labeler.yml

```diff
@@ -0,0 +1,25 @@
+name: Labeler
+
+on:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2.3.3
+
+      - name: Run Labeler
+        if: success()
```

Comment:
```suggestion
```

This seems redundant. If the previous step failed, the workflow aborts by default. See: 

- https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions